packages/l/llvm/pspec_x86_64.xml

```diff
@@ -839,245 +873,249 @@
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.fuzzer_interceptors.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.fuzzer_no_main.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.gwp_asan.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.lsan.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.profile.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.safestack.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.scudo.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.scudo.so</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.scudo_cxx.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.scudo_cxx_minimal.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.scudo_minimal.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.scudo_minimal.so</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.scudo_standalone.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.scudo_standalone.so</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.scudo_standalone_cxx.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.stats.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.stats_client.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.ubsan_minimal.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.ubsan_minimal.so</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.ubsan_standalone.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.ubsan_standalone.so</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/lib/i386-pc-linux-gnu/libclang_rt.ubsan_standalone_cxx.a</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/share/asan_ignorelist.txt</Path>
-            <Path fileType="library">/usr/lib32/clang/15.0.7/share/cfi_ignorelist.txt</Path>
+            <Path fileType="library">/usr/lib32/clang/16/include/__clang_cuda_builtin_vars.h</Path>
```

Comment:
There's a bunch of headers in here. Are these duplicates of the ones in the -devel package and if so can they be removed?

Reply:
The cuda ones and some randoms headers I picked are equivalent, but I don't know of an efficient way to make sure that _every_ file is equivalent. This is not urgent though, since it seems like these possibly duplicate headers have been there for a while.

Reply:
You could probably just copy the 32bit and 64bit headers to a different folder and run meld or diff against them. But agreed on it not being urgent, the build is complex enough that I'm not sure what to edit to even make them shared in the first place. I took a look at the Debian package and it looks like they have a shared package, but I'm not sure how their multiarch system even works in the first place so maybe the final location of the files on disk is still different.

I was moreso trying to look at the package with a fresh set of eyes since we're refreshing it anyway, having shared headers _would_ speed up builds that use the 32bit compiler since it would mean fewer files to be installed during the start of the build.